cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -834,7 +817,7 @@ spec:
             - name: CACHE_WARMING_ENABLED
               value: {{ (quote .Values.kubecostModel.warmCache) | default (quote true) }}
             - name: SAVINGS_ENABLED
-              value: {{ (quote .Values.kubecostModel.warmSavingsCache) | default (quote true) }}
+              value: "false"
```

Comment:
why force this to be false?

Reply:
I was about to type "because its the cost-model container" but I realize that 2.0 may release with one or two savings pages (RIs?) still served by cost-model. I'll revert this.

Reply:
on that note - do we also need to ensure this is passed to agg container? I think we still have savings cache there as well

Reply:
I'm trying to get away from the nested levels of defaulting and values, so I'm hoping we can just stick with the `kubecostAggregator.env` value map to set env vars like this.